packages/vscode-extension/package.nls.json

```diff
@@ -292,6 +292,11 @@
     "teamstoolkit.localDebug.output.tunnel.installSuccessMessage": "ngrok is installed at %s.",
     "teamstoolkit.localDebug.output.tunnel.skipInstallMessage": "Skip checking and installing ngrok as user has specified ngrok path (%s).",
     "teamstoolkit.localDebug.output.tunnel.createDevTunnelMessage": "Dev tunnel tag: %s.",
+    "teamstoolkit.localDebug.output.tunnel.deleteDevTunnelMessage": "Deleted Dev tunnel '%s'.",
+    "teamstoolkit.localDebug.output.tunnel.devTunnelLimitExceededMessage":"To continue debugging your Teams application, ensure all debugging Teams applications are closed and clean up the dev tunnels that were not properly removed by Teams Toolkit. Refer to [output channel](%s) to review tunnel details.",
```

Comment:
Exceeded dev tunnel limit. Clean up unused tunnels, close other debugging sessions and retry. Check [output channel](%s) for more details.

Reply:
Updated.